lms/djangoapps/courseware/views/index.py

```diff
@@ -412,9 +412,9 @@ def _create_courseware_context(self):
             'init': '',
             'fragment': Fragment(),
             'staff_access': self.is_staff,
-            'studio_url': get_studio_url(self.course, 'course'),
             'masquerade': self.masquerade,
-            'real_user': self.real_user,
```

Comment:
Yes, it was. I changed the template to get the real_user from the actual user, so that every view didn't have to pass through the real user.